data/json/items/ammo.json

```diff
@@ -303,6 +303,7 @@
         "damage" : 2,
         "range" : 10,
         "dispersion" : 14,
+        "loudness" : -999,
```

Comment:
To ensure they remained silent even if the weapon specified `loudness` but there isn't really a need to support that (and thinking about it such an approach wouldn't work anyway) so I've reverted this in 2976c39.